### ISSUE TITLE
Vulkan support?

### ISSUE THREAD

**Issue by cheako** [brackeen/glfm#17]
I'm looking for a multi-platform abstraction layer for Vulkan.

**Comment by brackeen** [comment]
Not at the moment. It could happen (with MoltenVK for iOS) but I don't have a timeframe for implementing it right now

**Comment by saidwho12** [comment]
Yeah feels bad, I was really looking forward to using this for my vulkan engine. I wish OpenGL had a better system for multi-threading.

**Comment by LDprg** [comment]
@brackeen how is it looking now. Vulkan support would be really nice, because it is nowadays more common than 5 years ago. Also it would allow game engines using glfw to add glfm for mobile (if the game engine does only support vulkan).

**Comment by cheako** [comment]
5yrs ago rust wasn't as viable an option as it is today.  Look at winit, it's overengineered and breaks APIs sometimes.  I use ash to access vulkan, but wgpu may be a better option and may even instantiate windows(replacing winit).

You can ask me more at oftc #dri-devel
